Makefile

```diff
@@ -31,6 +31,10 @@ core-test: generate
 	@echo "Testing core packages..." && \
 		go test -tags 'core' $(TESTARGS) $(shell go list ./... | grep -v -E 'terraform/(builtin|vendor)')
 
+get-deps:
```

Comment:
If these changes need to be included, would prefer a separate PR for adding to TF core.